emf/src/test/resources/properties.ecore

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ecore:EPackage xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" name="io.lionweb.Properties" nsURI="https://lionweb.io/io_lionweb_Properties"
+    xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" name="io_lionweb_Properties" nsURI="https://lionweb.io/io_lionweb_Properties"
```

Comment:
Wouldn't be better to keep the dots? Or is there a reason to replace them?

Reply:
invalid, see above

Reply:
Understood, thank you